cdap-ui/app/features/admin/controllers/namespace/metadata-ctrl.js

```diff
@@ -32,8 +32,7 @@ function ($scope, $state, myAlertOnValium, MyCDAPDataSource, myNamespace) {
     myAlertOnValium.show({
       title: 'It doesn\'t work yet',
       content: 'There is no content yet',
-      type: 'warning',
-      duration: false
+      type: 'warning'
```

Comment:
Do we want this? We should probably change the message. I think it was added initially when we want some metadata stuff in UI.